Gemfile

```diff
@@ -9,3 +9,5 @@ gemspec
 group :rubocop do
   gem "rubocop-shopify", require: false
 end
+
+gem "prism", github: "ruby/prism"
```

Comment:
Why can't we use the released version? Users will not know they need to point to a github branch.

Reply:
Just saw it.